lib/targets.go

```diff
@@ -157,6 +171,7 @@ func NewLazyTargeter(src io.Reader, body []byte, hdr http.Header) Targeter {
 		if err = sc.Err(); err != nil {
 			return ErrNoTargets
 		}
+		_ = sc.Peek()
```

Comment:
I think it would be a lot more reliable to consume all white space in the beginning of every invocation.
```go
		for sc.Scan() {
			if line := strings.TrimSpace(sc.Text()); line == "" {
				break
			}
		}
		tokens := strings.SplitN(line, " ", 2)
		if len(tokens) < 2 {
			return fmt.Errorf("bad target: %s", line)
		}
		switch tokens[0] {
		case "HEAD", "GET", "PUT", "POST", "PATCH", "OPTIONS", "DELETE":
```

That way we could also get rid of the `httpMethodChecker` regex.